docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/CMS_Alternating_Button.md

```diff
@@ -136,8 +136,12 @@ extras.
 Continuing our example let's add a "constructive" style to our *Clean-up* button. First you need to be able to add
 custom JS code into the CMS. You can do this by adding a new source file, here
 `mysite/javascript/CMSMain.CustomActionsExtension.js`, and requiring it
-through a YAML configuration value: `LeftAndMain.extra_requirements_javascript`.
-Set it to 'mysite/javascript/CMSMain.CustomActionsExtension.js'.
+through a YAML configuration value: 
+```
+<Name space>\<Class name>:
+  extra_requirements_javascript:
+    - mysite/javascript/<Name of file including including .js suffix
+```
```

Comment:
I think it might be good to clarify that this is a LeftAndMain configuration option, but (if this is true) that you can also apply it to subclasses of LeftAndMain. I think keeping LeftAndMain in the example code would be good, so you're just adding the namespace to the class

Reply:
I did not know it is a LeftAndMain specific thing.  Makes sense that it is and it makes sense to clarify that  (I am not a complete novice, I have promoted myself to beginner...)
I am applying it to a LeftAndMain descendant (via ModelAdmin) it is working nicely.

Reply:
Nice job! yes, this is LeftAndMain specific and it does apply to any classes that extends LeftAndMain :) Would be good to keep a mention of LeftAndMain (with the inheritance behaviour)

Perhaps an example for module files would be a good idea as well. Two extra lines like this:
```
# for module source files, you can do the following
    - silverstripe/blog:client/dist/js/main.bundle.js
```